lib/jxl/modular/transform/squeeze.cc

```diff
@@ -297,6 +301,9 @@ Status InvSqueeze(Image &input, std::vector<SqueezeParams> parameters,
     }
     for (uint32_t c = beginc; c <= endc; c++) {
       uint32_t rc = offset + c - beginc;
+      if (rc >= input.channel.size()) {
+        return JXL_FAILURE("Corrupted squeeze transform");
```

Comment:
I converted this to an assert(). It was because different list of transforms were passed to MetaSqueeze and InvSqueeze due to the different nb_meta_channels.